### DISCUSSION TITLE
Work on a new UI and UX

### DISCUSSION THREAD

**Discussion by puffdesigns** [deckerst/aves#291] [General]
Hello there,
First of all, I would like to say I found your app from a Reddit group, I am using it and it's cool man, thanks for building it. also, I want to contribute to the project, and being a UX designer I would like to help with a modern UI and improved UX if you are willing to let me help and work together. let me know. Thanks again.

**Comment by deckerst** [comment]
Hi, and thanks for your interest!

For now and the foreseeable future, I'd like to keep Aves as a personal project. This project does not accept PRs, except for translations. Same goes for UI/UX. For me it's part of the fun to design the UI and think about the UX.
